src/murmur/Cert.cpp

```diff
@@ -116,6 +117,27 @@ void Server::initializeCert() {
 		qlCA = ql;
 	}
 
+#if defined(USE_QSSLDIFFIEHELLMANPARAMETERS)
+	if (! dhparams.isEmpty()) {
+		QSslDiffieHellmanParameters qdhp = QSslDiffieHellmanParameters(dhparams);
+		if (qdhp.isValid()) {
+			qsdhpDHParams = qdhp;
+		} else {
+			QSet<QSslDiffieHellmanParameters::Error> errors = qsdhpDHParams.errors();
```

Comment:
Are we sure at least one of these two types is always there? Otherwise we might end up not using the parameters without the user being notified in the log. Is there some generic way to log all errors in the list (e.g. some error code)?